email-service/src/main/java/no/sikt/nva/email/EmailRequestHandler.java

```diff
@@ -49,6 +49,13 @@ public EmailRequestHandler() {
                 .withRegion(Regions.EU_WEST_1).build(), new Environment());
     }
 
+    @JacocoGenerated
```

Comment:
I JacocoGenerated needed here? I would assume the framework always call this method as part of request handling?

Reply:
yes it was super weird that I needed to put jacocogenerated here.